anymail/webhooks/unisender_go.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import annotations
+
+import json
+import typing
+from datetime import datetime, timezone
+from hashlib import md5
+
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse
+from django.utils.crypto import constant_time_compare
+
+from anymail.exceptions import AnymailWebhookValidationFailure
+from anymail.signals import AnymailTrackingEvent, EventType, RejectReason, tracking
+from anymail.webhooks.base import AnymailCoreWebhookView
+
+"""
+Callback API example:
+{
+   "auth":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+   "events_by_user":
+   [
+      {
+        "user_id":456,
+        "project_id":"6432890213745872",
+        "project_name":"MyProject",
+        "events":
+        [
+          {
+            "event_name":"transactional_email_status",
+            "event_data":
+            {
+              "job_id":"1a3Q2V-0000OZ-S0",
+              "metadata":
+              {
+                "key1":"val1",
+                "key2":"val2"
+              },
+              "email":"recipient.email@example.com",
+              "status":"sent",
+              "event_time":"2015-11-30 15:09:42",
+              "url":"http://some.url.com",
+              "delivery_info":
+              {
+                "delivery_status": "err_delivery_failed",
+                "destination_response": "550 Spam rejected",
+                "user_agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36...",
+                "ip":"111.111.111.111"
+              }
+            }
+          },
+          {
+            "event_name":"transactional_spam_block",
+            "event_data":
+            {
+              "block_time":"YYYY-MM-DD HH:MM:SS",
+              "block_type":"one_smtp",
+              "domain":"domain_name",
+              "SMTP_blocks_count":8,
+              "domain_status":"blocked"
+            }
+          }
+        ]
+      }
+   ]
+}
+"""
+
+
+class UnisenderGoTrackingWebhookView(AnymailCoreWebhookView):
+    """Handler for UniSender delivery and engagement tracking webhooks"""
+
+    esp_name = "UnisenderGo"
+    signal = tracking
+
```

Comment:
Probably want to set `warn_if_no_basic_auth = False`, since MD5 signature validation is sufficient.

(Example of [webhook that uses signature verification](https://github.com/anymail/django-anymail/blob/main/anymail/webhooks/mailgun.py#L36-L62).)